### PR TITLE
Add rules page

### PR DESCRIPTION
**Pull request by sah-shah7**
-Added Rules page to the Quiz App
image

### CODE REVIEW

File: stylesheets/shared/layout.css [hunk 1]
```diff
@@ -30,6 +30,9 @@
     padding-top: 1rem;
     padding-bottom: 1rem;
 }
+.pd-x-t-1{
```
Comment (zabihshaik) [inline stylesheets/shared/layout.css:33]: Better to use as less as possible. For eg, it can be .pd-t-1 ig

File: stylesheets/shared/layout.css [hunk 2]
```diff
@@ -38,9 +41,13 @@
     margin-right: -15px;
```
Comment (zabihshaik) [inline stylesheets/shared/layout.css:41]: Better to use shorthand property

**Review by zabihshaik — APPROVED** [review]
Everything looks good. Just a couple of comments.

File: src/pages/category-1-pages/rules.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Sahil Shah" />
+    <link rel="stylesheet" href="../../../stylesheets/shared/shared.css" />
+    <link rel="stylesheet" href="../../../stylesheets/home.css" />
+    <link rel="stylesheet" href="../../../stylesheets/rules.css" />
+    <link rel="stylesheet" href="../../../stylesheets/quiz-category.css">
```
Comment (sharath-io) [inline src/pages/category-1-pages/rules.html:12]: Can maintain a css file and import all other css files required.

**Review by sharath-io — APPROVED** [review]
Looks good to me. Can merge. Good to use import for css files into one file.

File: stylesheets/shared/layout.css (hunk 1 above)
Comment (sah-shah7) [reply stylesheets/shared/layout.css:33]: Follow the x & y direction naming scheme

File: stylesheets/shared/layout.css (hunk 2 above)
Comment (sah-shah7) [reply stylesheets/shared/layout.css:41]: Noted

**Comment by sah-shah7** [comment]
PR approved. Merging

**Pull request merged**